.eslintignore

```diff
@@ -0,0 +1,6 @@
+app/bower_components
+node_modules
+.tmp
+dist
+tmp
+test/integration
```

Comment:
Pourquoi ignorer `test/integration` ? Peux-tu ajouter un commentaire ?

Reply:
J'ai simplement copié ce qui était dans `.jshintignore`

Reply:
Ok ça pète sévère dans `test/integration` à cause des fichiers js pour watai.

Reply:
Il y a une grosse duplication, que penses-tu de deux blocs conditionnels portant juste la gestion des fichiers statiques ?

`➜  mes-aides-ops git:(nginx-static) diff /home/thomas/repos/mes-aides-ops/modules/mesaides/templates/mesaides_config.erb /home/thomas/repos/mes-aides-ops/modules/mesaides/templates/nginx_config.erb
80,88d79
<   <% if defined? @nginx_root %>
<   root <%= @nginx_root %>;
<   <% end %>
< 
<   include snippets/mes-aides-static.conf;
< 
<   # WARNING
<   # Everything in app/ or dist/ will be served directly by Nginx
<   # Do not put any sensitive file there
90,93d80
<     try_files /dist$uri /app$uri @default;
<   }
< 
<   location @default {
```